multimodal-search-tirg/pods/query-text-filter.yml

```diff
@@ -0,0 +1,8 @@
+!BaseExecutor
+requests:
+  on:
+    SearchRequest:
+      - !FilterQL
+        with:
+          traversal_paths: ['c']
+          lookups: {'modality': 'text'}
```

Comment:
same here, not sure if the `lookups` syntax is correct, to be checked

Reply:
Why does [this](https://github.com/jina-ai/examples/blob/07d09f6a040da5ec200e136766d53e72946ae46f/cross-modal-search/yaml/text_encoder.yml) use different filter lookups for index and query? Is it because of query coming from jinabox?

Reply:
you caught it! !

It is actually a dirty hack to make it work with `jinabox` but it should come from `modality`

Reply:
When we move to work with `jinabox` we may do the same trick here, but at least we should know is a dirty hack